app/views/sign_up/completions/show.html.slim

```diff
@@ -1,10 +1,10 @@
--title t("titles.loa3_verified.#{session[:sp][:loa3]}", app: APP_NAME)
+-title t("titles.loa3_verified.#{session['sp][:loa3']}", app: APP_NAME)
```

Comment:
this was translated wrong, since there are two symbols, it needs to be:

```
session['sp']['loa3']
```

but like I mentioned earlier, I think we should just do `.with_indifferent_access`